hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/compress/TestCodec.java

```diff
@@ -109,7 +109,7 @@ public void testGzipCodec() throws IOException {
     }
     // without hadoop-native installed.
     ZlibFactory.setNativeZlibLoaded(false);
-    assumeTrue(ZlibFactory.isNativeZlibLoaded(conf));
+    assertFalse(ZlibFactory.isNativeZlibLoaded(conf));
```

Comment:
@sunchao I found two places wrongly use `assumeTrue`. It should be `assertFalse`. Since it is two line change, I also include here.